src/main/java/org/openpnp/gui/tablemodel/BoardLocationsTableModel.java

```diff
@@ -33,13 +33,19 @@
 
 public class BoardLocationsTableModel extends AbstractTableModel {
     private final Configuration configuration;
-
+    public enum BoardStatus {
+        Ready,
+        MissingPart,
```

Comment:
MissingPart, MissingFeeder and ZeroPartHeight aren't used here as far as I can tell. They should be removed from the enum if they aren't used.

Reply:
Right, will add just "Error" instead indicating that there are unresolved issues to deal with in the future.